starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -465,6 +465,13 @@ public void onError(MediaCodec codec, MediaCodec.CodecException e) {
             }
           }
 
+          @Override
+          public void onCryptoError(MediaCodec codec, MediaCodec.CryptoException e) {
+            synchronized (this) {
+              Log.e(TAG, "onCryotpError: " + e.getErrorCode());
```

Comment:
If we use `CONFIGURE_FLAG_USE_CRYPTO_ASYNC`, we need to notify the starboard player if there's any crypto error.